requirements/edx/testing.txt

```diff
@@ -1372,19 +1390,24 @@ user-util==1.0.0
     # via -r requirements/edx/base.txt
 uvicorn==0.15.0
     # via pact-python
-vine==1.3.0
+vine==5.0.0
     # via
     #   -r requirements/edx/base.txt
     #   amqp
     #   celery
-virtualenv==20.10.0
+    #   kombu
+virtualenv==20.8.1
```

Comment:
Just noting downgrade. Not sure if this is anything to be concerned about, but just bringing it to your attention.